config/default-example.yaml

```diff
@@ -11,6 +11,11 @@ authentication:
   username: ${API_USER}
   password: ${API_PASSWD}
 
+pagination:
```

Comment:
Should this be configurable? Whether an api is paginated, it's max page size, and the default page size will depend on the capabilities of the backend vendor. For example, if a vendor API only supports a maximum of 100 results per page, we would want to hardcode that rather than have it come from a configuration file, since that number would be the same regardless of the environment.